src/main/tv/codely/scala_intro_examples/lesson_10_enums_vs_sealed_structures/exercise_solutions/daniel_albarral/GenericPicoteriaProduct.scala

```diff
@@ -0,0 +1,18 @@
+package tv.codely.scala_intro_examples.lesson_10_enums_vs_sealed_structures.exercise_solutions.daniel_albarral
+
+import java.util.UUID
+
+trait PicoteriaProduct {
+  val id: UUID
+}
+
+trait GenericFood extends Ordered[GenericFood] {
+
+  override def compare(that: GenericFood): Int = that.getForcefulness - getForcefulness
+
+  val getForcefulness: Int
+
+  def getFatLevel: String
```

Comment:
Why declaring it as `def` instead of `val`?

If we declare it as a `val` we would avoid having to evaluate it on each call :)